src/vacuum-card.js

```diff
@@ -351,9 +355,14 @@ class VacuumCard extends LitElement {
 
   render() {
     const { state } = this.entity;
-    const { status, battery_level, battery_icon } = this.getAttributes(
-      this.entity
-    );
+    const {
+      status,
+      battery_level,
+      battery_icon,
+      valetudo_state,
+    } = this.getAttributes(this.entity);
+
+    const val_status = status || valetudo_state['name'];
```

Comment:
Can we hide this platform-specific properties, like `valetudo_state` under `getAttribute` method?

Reply:
Good point. Done.